device-data/types/basal/suppressed.md

```diff
@@ -1,5 +1,279 @@
 <!-- non-generated document! all areas editable -->
 
-Coming soon!
+## the `suppressed` field on `temp`s and `suspend`s
 
-<!-- TODO: document how `suppressed` works when a `temp` or `suspend` basal crosses > 1 segment of a basal schedule -->
\ No newline at end of file
+Some insulin pump data protocols provide enough information for us to track various aspects (e.g., `rate`, `scheduleName`) of the basal that *would have been in effect* had the currently active `temp` or `suspend` basal **not** been programmed (or triggered). Where this information is available, we provide it in as much detail as possible as an embedded object in the `suppressed` field on a `temp` or `suspend` basal interval.
+
+In general, this `suppressed` object need and **must** only contain the bare minimum of information:
+- the `type` (= `basal`)
+- the `deliveryType` (only `scheduled` if the currently active basal is a `temp`; otherwise (i.e., if the currently active basal is a `suspend`), may be `scheduled` *or* `temp`)
+- the `rate` of insulin delivery
+- (optionally) the `scheduleName` if the suppressed interval is a `scheduled` basal and the name of the basal schedule is available through the insulin pump's data protocol[^a]
+
+If the currently active basal is a `suspend` and the `suppressed` is a `temp`, then the following `temp` fields may also be present on the `suppressed` object:
+- the `percent` of the scheduled basal rate
+- another(!) nested `suppressed` object representing the originally `scheduled` basal interval (see [below](#the-possibility-of-nested-suppressed-in-suspend-basals) for details)
+
+(More details of the precise allowed shape of a nested `suppressed` are available in the [`temp` basal documentation](./temp.md#suppressed)) and the [`suspend` basal documentation](./suspend.md#suppressed).)
+
+Note in particular that we do **not** include any timestamp or duration information in the `suppressed`: by definition, these values are always equal to those of the *active* basal interval's, and so it is not necessary to specify them.
```

Comment:
This is not true with actual production data. See https://gist.github.com/darinkrauss/7a91115d0d3475e4b11e559bf6dcdcaf

Also, this isn't just legacy data; it is data going in now.